src/components/achievements/Achievements.js

```diff
@@ -1,13 +1,17 @@
-import React from 'react';
-import CardHeader from '~/components/cardheader/CardHeader';
-import EditOptions from '~/components/editoptions/EditOptions';
 import { Add } from '~/assets/image';
+import React, {useState} from 'react';
+import EmptyCard from '../emptycard/EmptyCard';
+import CardHeader from '~/components/cardheader/CardHeader';
 import CardFooter from '~/components/cardfooter/CardFooter';
+import EditOptions from '~/components/editoptions/EditOptions';
 
 const Achievements = () => {
+	const [isDataAvailabel, setData] = useState(true);
```

Comment:
I guess you meant to write `isDataAvailable` instead of `isDataAvailabel`?

Reply:
🆗